app/views/request/new.erb

```diff
@@ -0,0 +1,7 @@
+<form action="/request/new" method="post">
```

Comment:
This might not be needed now as we have the calendar functionality, so we do not need to manually input dates.